_pages/business-units/18f/engineering-resources/before-you-ship/security.md

```diff
@@ -0,0 +1,289 @@
+---
+title: Security
```

Comment:
I would put a permalink on this to be explicit. I was surprised this ended up at /security/ by default.

```suggestion
title: Security
permalink: security/
```

Reply:
Sure thing -- all the Markdown files appear at the top-level by default (as Aidan mentioned this morning, part of my work is to think about whether this is a good default or not).